examples/codegen/int_string_cast.an

```diff
@@ -0,0 +1,31 @@
+max_u8: u8 = 255
+
+smu: string = cast max_u8
+
+print <|
+  "string: " ++
+  smu ++
+  " len: " ++
+  cast smu.length
```

Comment:
Lets put this printing in a helper function to avoid repeating it. I think it'd help to put the focus more on the int to string cast rather than the debug printing which currently takes up most of the file